Add HILDA 2023 website to the repo

behroozomidvar opened the pull request.

behroozomidvar committed:
Create 2023 folder

behroozomidvar committed:
Upload files

behroozomidvar committed:
Delete init.txt

domoritz commented:
Please update the previous years as well and link to 2023.

domoritz reviewed (changes requested):
* Please update the previous years as well and link to 2023. 
* Update the colors for 2023
* Update the text since the format is not new in 2023 but was in 2022
* Update the submission link to the right platform

behroozomidvar committed:
Change theme color for 2023

behroozomidvar committed:
Change intro content

behroozomidvar committed:
Set submission website to TBD for now. 

CMT link is create but we'd like to review the policies precisely before sharing it on the website.

behroozomidvar committed:
Redirect to 2023
